packages/messenger-web/src/Dm3Widget.tsx

```diff
@@ -1,17 +1,56 @@
-// Import React and the CSS for this component
 import React from 'react';
 import './Dm3Widget.css';
 
-// Ignore TypeScript errors for the next line
-//@ts-ignore
-// Import the DM3 component from the 'messenger-widget' package
+// Custom hook to manage message destination
+import { useMessageTo } from './useMessageTo';
+// Importing the DM3 component and its configuration type from the DM3 Messenger Widget package
 import { DM3, DM3Configuration } from '@dm3-org/dm3-messenger-widget';
 
-// Define the Dm3Widget component
+/**
+ * Helper function to generate a weekly-based pseudo-random image path.
+ * It uses the current week of the year to ensure the image remains consistent throughout the week,
+ * but changes weekly.
+ * @returns The path to the selected image.
+ */
+const getWeeklyImagePath = (): string => {
```

Comment:
To make the code cleaner please move this function to a separate file **utils/weekPath.ts** and call this here. May be in future we have multiple utility functions and that could make the file look longer.